File: test44_out/test44/BlinkEnvPin/BlinkEnvPin.c

/*
 * ======== string literals ========
 */

static const char test44_BlinkEnvPin__s__fileName[] = "BlinkEnvPin.p";
static const char test44_BlinkEnvPin__s__0[] = "BlinkEnvPin";

/*
 * ======== function definitions ========
 */

void test44_BlinkEnvPin_pollen__reset__E() {
    atmel_atmega8_PB5_clear__E();
}

void test44_BlinkEnvPin_pollen__run__E() {
    uint16 i;
    
    while (true) {
        atmel_atmega8_PB5_toggle__E();
        for ((i) = 0; (i) < 250; (i)++) {
            atmel_atmega8_Core_wait__E(1000);
        }
    }
}

void test44_BlinkEnvPin_targetInit__I() {
}

File: test44_out/test44/BlinkEnvPin/BlinkEnvPin.js

/*
 * ======== MODULE test44.BlinkEnvPin ========
 */

var $$c = function() {
    const BlinkEnvPin = {};
    BlinkEnvPin.BlinkEnvPin = BlinkEnvPin
    BlinkEnvPin.$name = 'test44.BlinkEnvPin';
    BlinkEnvPin.pollen$used = 0;
    BlinkEnvPin.pollen__reset = new $$Ref('test44_BlinkEnvPin_pollen__reset__E');
    BlinkEnvPin.pollen__run = new $$Ref('test44_BlinkEnvPin_pollen__run__E');
    BlinkEnvPin.targetInit = new $$Ref('test44_BlinkEnvPin_targetInit__I');
    BlinkEnvPin.$$hostInit = function() {
        var $$text = '';
        return $$text;
    }
    BlinkEnvPin.pollen__uses__ = function() {
        $$bind($units['atmel.atmega8.Core'], 'pollen$used', true);
        $$bind($units['atmel.atmega8.PB5'], 'pollen$used', true);
        $$bind($units['pollen.hardware.Led'], 'pollen$used', true);
    }
    BlinkEnvPin.$$privateInit = function() {
    }
    return BlinkEnvPin;
}
var $$u = $$c();
$units['test44.BlinkEnvPin'] = $$u;
$units.push($$u);

